### DISCUSSION TITLE
Update of the script in the popup

### DISCUSSION THREAD

**Discussion by jojoPL1987** [violentmonkey/violentmonkey#1569] [Ideas]
Sometimes I need to update one script that is already running on a page when I have that page open. Then I have to open the Violentmonkey popup, click the gear icon and then update all scripts or find the one specific to update it manually.

The popup next to the script has a menu (in the button with three vertical dots) with options: "Homepage", "Exclude ..." and "Remove". Maybe add "Check for updates" to this menu?

**Comment by tophf** [comment]
Implemented in 7d10610f13a15c0a173c45659fdb1bbcf8eb05b6.

**Reply by gera2ld** [reply]
Looks like everything will be moved to the popup. 😆
